tests/utils/performance.ts

```diff
@@ -0,0 +1,31 @@
+export async function getLoadingDurations() {
```

Comment:
Like github says, this is a ts file so you're missing the return type (and myabe other ts stuff?)